lang/csharp/src/apache/main/Reflect/README.md

```diff
@@ -106,6 +106,40 @@ _Example TypedFieldConverter_:
     }
 ```
 
+### Nullable primitive properties
+
+Converters for primitive types (int, long, decimal, etc) automatically work with simple nullable unions/properties with the converters defined for the 
+non-nullable type.  You can create one converter and it will work for both nullable and non-nullable properties.
+
+```csharp
+        public class StringToDecimalConverter : TypedFieldConverter<string,decimal>
```

Comment:
Awesome add - I wish we could bubble this to the top of the PR

Reply:
I wasn't sure what their policy was for including helper classes.  I think _some_ people use Fixed for decimals (a byte array) instead of string.  I prefer string.  Others might prefer Fixed.  So I wasn't sure if the library should have an opinion on it.  But I would game for adding helper converters like this - AND the Fixed so that there isn't an opinion, just a sampling of common converters.